Routes/tasks-routes.js

```diff
@@ -0,0 +1,153 @@
+module.exports = function(app) {
+    const {v4: uuidv4} = require('uuid')
+    const db = require('C:/Users/Matan/task-manager-server/db.js')
+
+    const supportedFields = ['title', 'priority']
+    const lastUpdatedTime = 'lastUpdatedTime'
+    const createTime = 'createTime'
+    const id = 'id'
+
+    app.get('/api/tasks/:id', async function(req, res) {
+        try {
+            var data = await db.readData()
+        }
+        catch (e) {
```

Comment:
if the catch is for general/unexpected case so its better to do the try catch for wrapping all the lines in the function/api so if something is break in line 19 for example it will return your message instead the real error